scan_filter_chain:
- name: shadows
  type: ScanShadowsFilter
  params:
    min_angle: 10
    max_angle: 170
    neighbors: 20
    window: 1
#- type: LaserScanFootprintFilter
#  name: footprint_filter
#  params:
#    inscribed_radius: 0.4
